src/plugins/api-proxy/index.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import fs from 'fs';
+import util from 'util';
+import { duotones, getDuotoneUrls } from './util/duotone';
+import getApiProxyRoutes from './routes';
+import proxyApi$ from './proxy';
+
+const API_PROXY_PLUGIN_NAME = 'api-proxy';
+const API_ROUTE_PATH = '/mu_api';
```

Comment:
it definitely could be - I've basically set this as the plugin's default `path`, but it can be overridden by passing in `path` in the plugin options. I think I would lean toward keeping the default defined here, because it's unlikely to change and it feels simpler to avoid requiring an explicit setting to be passed down from the consumer. The downside is that it's a little hidden.

It would be pretty simple to add it to mwp-cli/src/config where we have most of our app config rules defined, but I almost feel like putting it there would make it easier to accidentally change and break stuff?